project-reports-lib.ts

```diff
@@ -75,6 +76,58 @@ export function getStringFromLabel(card: ProjectIssue, re: RegExp): string {
   return str
 }
 
+export function getLastCommentField(
+  issue: ProjectIssue,
+  field: string
+): string {
```

Comment:
Does it make sense to return `string | null` here so that it's clearer to a reader that there may be no last comment? That way a clearer check of `if lastComment == null` can be done.

Reply:
good suggestion.  will do